jabgui/src/main/java/org/jabref/gui/slr/ManageStudyDefinitionViewModel.java

```diff
@@ -119,6 +129,47 @@ public ManageStudyDefinitionViewModel(@NonNull Study study,
         this.directory.set(studyDirectory.toString());
         this.workspacePreferences = workspacePreferences;
         this.dialogService = dialogService;
+
+        initializeValidationBindings();
+    }
+
+    private void initializeValidationBindings() {
+        titleValidationMessage.bind(Bindings.when(title.isEmpty())
```

Comment:
Please use our ValidationSatus approach with the controlsfx visualizer 
https://github.com/JabRef/jabref/blob/3b6279510ed1e64bf63f15d57b591d23f4689088/jabgui/src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java#L138-L174

PS: Next time tell your AI tool to follow **existing patterns** in the codebase